docs/source/changelog.rst

```diff
@@ -30,6 +30,7 @@ Changelog
         * Remove unused variance_selection.py file (:pr:`613`)
         * Remove Timedelta data param (:pr:`619`)
         * Remove DaysSince primitive (:pr:`628`)
+        * Moved dask, distributed imports (:pr:`634`)
```

Comment:
move change long entry to Future Release

Reply:
Fixed.